### ISSUE TITLE
Apply fine tuned model to some images.why set is_training=True

### ISSUE THREAD

**Issue by zhoulinyuan** [tensorflow/models#2047]
in slim_walkthrough.ipynb
Apply fine tuned model to some images.
    # Create the model, use the default arg scope to configure the batch norm parameters.
    with slim.arg_scope(inception.inception_v1_arg_scope()):
        logits, _ = inception.inception_v1(images, num_classes=dataset.num_classes, is_training=True)
why set is_training=True, and don't set FALSE

**Comment by reedwm** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
